internal/controller/apiextensions/definition/reconciler.go

```diff
@@ -267,8 +282,9 @@ func NewReconciler(mgr manager.Manager, opts ...ReconcilerOption) *Reconciler {
 
 // A Reconciler reconciles CompositeResourceDefinitions.
 type Reconciler struct {
-	client resource.ClientApplicator
-	mgr    manager.Manager
+	client    resource.ClientApplicator
```

Comment:
Does this still need to be a ClientApplicator? We don't call the Apply method anymore, right?